### ISSUE TITLE
DISARM ability deletes items

### ISSUE THREAD

**Issue by bdagenais2** [GC-spigot/AdvancedEnchantments#2707]
### Describe the bug

If item in player's hand cannot be enchanted, item gets deleted instead of being moved or dropped.

### How to reproduce

Hold enderpearls, goldenapples, or dirt, and get hit with an enchant that uses DISARM.

### "/ae plinfo" link

https://paste.md-5.net/pirugufedo

**Comment by Burchard36** [comment]
Hello, can you send the current configuration you are using for this enchantment?

**Comment by bdagenais2** [comment]
> Hello, can you send the current configuration you are using for this enchantment?

```disarm:
  display: "%group-color%Disarm"
  description: "Chance to disarm opponent."
  applies-to: "Swords"
  type: "ATTACK"
  group: "FABLED"
  applies:
    - ALL_SWORD
  levels:
    1:
      chance: 2
      cooldown: 20
      effects:
        - "DISARM %victim%"
    2:
      chance: 4
      cooldown: 20
      effects:
        - "DISARM %victim%"
    3:
      chance: 6
      cooldown: 20
      effects:
        - "DISARM %victim%"```

**Comment by Burchard36** [comment]
Okay wow, this issue was HARD to fix, but not because of the complexity of the code but just because the bug description itself

So basically: It doesnt matter what the item was, it mattered what slot the players held item was in, if the player was holding the slot in slot 0 (First item in the hotbar) the item got removed however if they were holding any other slot then this worked perfectly fine

This bug will be fixed when 8.9.11 is released, if you can next time you open a bug report please send a video with it especially something like this :) It makes our lives a bunch easier as it took me a very long time to figure out that slot 0 was the real culprit here hehe

Not sounding rude here if I am I sincerely apologize, but when bugs are very specifically wrote out as well as videos, it makes our live a lot easier and we get the bug fixed a lot faster :)

Hope you have a nice day!

**Comment by Burchard36** [comment]
8.9.11 is out now! Please update to the latest version :)

If you have any new issues please open a new issue request!
